velodyne_pointcloud/cfg/TransformNode.cfg

```diff
@@ -33,7 +33,13 @@ gen.add("view_width",
 gen.add("frame_id", 
   pgc.str_t, 
   0, 
-  "new frame of reference for point clouds", 
+  "new frame of reference for point clouds",
+  "velodyne")
+
+gen.add("fixed_frame_id",
+  pgc.str_t,
+  0,
+  "inertial reference frame for point clouds",
```

Comment:
Would you have a problem changing these to "frame_id" (as to not disturb current software) and "intertial_frame_id"? I think using the term "fixed" is sort of a misnomer since the "inertial frame" can also change relative to the "world frame" and I want to make sure people don't confuse the two.

Reply:
I don't have a problem with making that change. However, I think is labelled as fixed_frame_id due to pcl_ros transformPointCloud calling the parameter fixed_frame with the description "fixed TF frame".

Since the inertial frame can move around, this makes me wonder if there is a potential bug here?

Reply:
See new changes and top level comment